src/content/getting started/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Getting started
+description: How do I start to set up the platform. A clear guide how to start to work with us properly sorted and documented.
```

Comment:
This description gets rendered, and should be completely reworded. If this was meant to be done after this PR, I'm fine.

[repeated below]

Reply:
if you can plz review now